omicron/cli/process.py

```diff
@@ -1030,8 +1037,11 @@ def main(args=None):
     ppjob.add_condor_cmd('+InitialRequestMemory', f'{ppmem}')
     ppjob.add_condor_cmd('request_memory',
                          f'ifthenelse(isUndefined(MemoryUsage), {ppmem}, int(3*MemoryUsage))')
+    ojob.add_condor_cmd('allowed_job_duration', 3*3600)
```

Comment:
Lint says, `./omicron/cli/process.py:1040:50: E226 missing whitespace around arithmetic operator`, but I'm not sure what the issue is.  Isn't the standard to use spaces around "+" and "-" but not around "*" and "/"?

Reply:
done